imgp

```diff
@@ -585,7 +603,14 @@ def main():
     eraseexif = args.eraseexif
     force = args.force
     includeimgp = args.includeimgp
-    keep = args.keep
+    if args.keep:
+        keep = True
+        if 'x' in args.keep:
+            khres, kvres = getres(args.keep)
+        else:
+            khres = kvres = try_int(args.keep)
+    else:
```

Comment:
No.

I meant if the call is:

    imgp -k
    // -k without any arguments

khres should be set to HRES and kvres should be set to VRES.